packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -1081,6 +1081,7 @@ function BudgetHeader({
             // zIndex: -1
           },
         ]}
+        data-vrt-mask
```

Comment:
oh that's super cool -- I wondered how you were marking out those areas in the snapshots. 👍🏻